tracetools_trace/tracetools_trace/trace.py

```diff
@@ -79,6 +83,15 @@ def _resolve_session_path(
     print(f'writing tracing session to: {full_session_path}')
     return base_path, full_session_path
 
+def _resolve_session_live_url(
+    *,
+    session_name: str,
+) -> str:
+    live_tracing_url = (
+        'net://localhost/host/' + socket.gethostname() + '/' +
+        session_name
+    )
+    print(f'live trace data will be sent to: {live_tracing_url} on the system running lttng-relayd')
```

Comment:
I would remove "on the system running lttng-relayd" but still mention the relay daemon before the `:`, like:

```suggestion
    print(f'live trace data will be sent to the relay daemon at: {live_tracing_url}')
```

Reply:
Done